docs/user-guide/global-configurations/cluster-and-environments.md

```diff
@@ -108,15 +98,45 @@ Now, click `Save Cluster` to save your cluster on Devtron.
 
 Your Kubernetes cluster gets mapped with Devtron when you save the cluster configurations. Now, the Devtron agent must be installed on the added cluster so that you can deploy your applications on that cluster.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/install-devtron-agent.png)
+![Figure 8: Installing Devtron Agent](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/install-devtron-agent.png)
```

Comment:
Let's remove devtron agent from doc, wherever it is mentioned, it is not required now, devtron agent has retired